tests/test_slim_engine.py

```diff
@@ -1473,6 +1492,7 @@ def test_drawn_mutation_meets_AF_threshold(self):
                 seed=seed,
             )
             assert ts.num_mutations == 1
+            ts = pyslim.SlimTreeSequence(ts)
```

Comment:
we only need pyslim treeseq in this test because it uses `individuals_alive_at`.